docs/FEATURES.md

```diff
@@ -0,0 +1,100 @@
+# Features
```

Comment:
Very nice doc

Reply:
Thank you. Was about to ask if this reflects your thinking.